### DISCUSSION TITLE
Integration of AWS Glue Workflows Support

### DISCUSSION THREAD

**Discussion by mkhalid12** [vincentclaes/datajob#137] [Ideas]
This is such a easy to use library to deploy the workflow. However, is there a possibility to create an [AWS Glue Workflow](https://docs.aws.amazon.com/glue/latest/dg/workflows_overview.html) for glue Jobs in a similar way, this library helps to create Step Funcitons.

**Comment by vincentclaes** [comment]
Hey, thanks!

Glue workflow is not supported. Stepfunctions has a much better integration and advanced functionality within the entire AWS ecosystem.
Everything is written in AWS CDK, so you can just use the Glue Workflow constructs: https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_glue/CfnWorkflow.html

Hope that helps!

Vincent
